build/flaws.js

```diff
@@ -485,4 +506,21 @@ async function fixFixableFlaws(doc, options, document) {
   }
 }
 
+function getImageminPlugin(fileName) {
+  const extension = path.extname(fileName);
+  if ((extension === ".jpg") | (extension === ".jpeg")) {
```

Comment:
Wait is it just me or is `or` written `||` and not just a single pipe?

Reply:
That's an excellent find!

Reply:
Updated now.